### DISCUSSION TITLE
about forcing everyone to use 2FA authentication

### DISCUSSION THREAD

**Discussion by rodrigo1593** [community/community#66131] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

limiting their capability to do precise things is not good for us people who doesn't have any phone number available or cellphones, or also could be lost/taken easily if we are in dangerous locations.

so i would like to have the posibility to decide if i want to have my projects more safe or not. bc there is still disadvantages no matters what of those two i take on.

or either, another kind of 2FA that doesnt require phone number. please.

**Comment by supervisitor** [comment]
Whether you don't have a cell phone or you don't want or are not allowed to use the number (business phone), GitHub/MS wants it. 😉
The only thing that helps is to [migrate to alternatives](https://github.com/orgs/community/discussions/49715#discussioncomment-6928416), [contact the support directly](https://github.com/orgs/community/discussions/65344#discussioncomment-6929870) and explain the [real reasons and justified doubts](https://github.com/orgs/community/discussions/49715#discussioncomment-6728834) about the technical sense.
